src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -203,6 +207,11 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
       (s) => s.index > step.index && s.isVisited
     );
 
+    // Only this code is different from the original
```

Comment:
I've pulled this up to make it clear what's the difference with the original code :)